GameBox/GameBox/ECS/Systems/AISystem.h

```diff
@@ -194,6 +198,11 @@ class AStarPathfinding
 		}
 	};
 
+	// Used to get the distance between nodes.
+	sf::Vector2f getNodeRelativeDistance() { 
```

Comment:
Consider moving this function to a separate math helper header so it can be reused at other places